.github/workflows/macos.yml

```diff
@@ -13,6 +13,21 @@ on:
     # https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks
   merge_group:
 
+env:
+  # GITHUB_PULL_REQUEST_URL are used for commenting test reports in Launchable Github App.
+  # https://github.com/launchableinc/cli/blob/v1.80.1/launchable/utils/link.py#L42
+  GITHUB_PULL_REQUEST_URL: ${{ github.event.pull_request.html_url }}
```

Comment:
I don't understand why when setting the environment variable `GITHUB_PULL_REQUEST_URL` here, Launchable CLI can get the value from the environment variable `GITHUB_PULL_REQUEST_URL_KEY`? These environment variables are different.

Reply:
The value of `GITHUB_PULL_REQUEST_URL_KEY` is `GITHUB_PULL_REQUEST_URL`.

Reply:
Thus, Launchable CLI can get this value.

Reply:
Ah right! Thanks for explaining it.